charts/orchestrator/values.yaml

```diff
@@ -34,23 +34,55 @@ postgres:
   dataDir: /var/lib/postgresql/data/dbfiles  # where the database data files are stored inside the container
 
 # override Janus image with one with the orchestrator plugin
-backstage: 
+backstage:
+  global:
+    dynamic:
+      includes:
+        - dynamic-plugins.default.yaml
+      plugins:
```

Comment:
do we want to add notification plugin here?

Reply:
when it will be released and ready to install on showcase. will not be added in this version of the PR, but in the roadmap to be added.